Plants Redone

DragonSlayer62 opened the pull request:
- [ADD] Added new NPC definition for Treefellow in plant_base.dfn, with _t2a, _lbr and _aos variants in respective files
- [ADD] Added new NPC definitions for Doom Tree and Death Willow in agressiveplants.dfn
- [UPD] Renamed agressiveplants.dfn to aggressiveplants_base.dfn and redefined NPC entries within as base definitions, with _lbr and _aos variants in respective files

DragonSlayer62 committed:
Plants Redone

- [ADD] Added new plant_base.dfn with treefellow npc
- [ADD] Added Unique Creatures Extinct Plants
- [UPD] Updated agressiveplants.dfn as base and created agressiveplants lbr and aos dfn files.

DragonSlayer62 committed:
Merge branch 'develop' into agressiveplant-update

strattond commented on data/dfndata/npc/aggressiveplants_base.dfn:
```diff
@@ -8,222 +8,235 @@
+GETTOL=bogthing_aos
+}
+
+[base_bogling]
```
This doesn't pull from base_plants so it loses DIRECTION, BACKPACK, NPCWANDER, and FX2.  The FLAG can be omitted if pulled from base_plants.

strattond commented on data/dfndata/npc/aggressiveplants_base.dfn:
```diff
@@ -8,222 +8,235 @@
+}
+
+[base_quagmire]
 {
```
No GET for either base_plants or base_aggressiveplants

strattond commented on data/dfndata/npc/aggressiveplants_base.dfn:
```diff
@@ -8,222 +8,235 @@
+}
+
+[base_bogthing]
 {
```
No GET from base_plants or base_aggressiveplants

strattond reviewed (changes requested):
Looks reasonable, couple of missed opportunities I believe

DragonSlayer62 committed:
couple small issues

fixed couple small issues requested

Xoduz committed:
Minor update

Renamed agressiveplants.dfn to aggressiveplants_base.dfn
Renamed agressiveplants_aos.dfn to aggressiveplants_aos.dfn
Renamed agressiveplants_lbr.dfn to aggressiveplants_lbr.dfn
Added flag (good), and updated race (18->19) for base_trefellow
Fixed incorrect magic resistance skill entry and damage stat for treefellow_aos
- [ADD] Added plants_lbr.dfn and plants_t2a.dfn with Treefellow definitions for those eras, and updated base Treefellow definition to point to these for the respective eras

Xoduz committed:
Minor stat updates to aggressiveplants

Minor stat updates to aggressiveplants
